sync/linux/controlplane.sh

```diff
@@ -40,7 +32,7 @@ fi
 
 kubernetes_version=${1}
 k8s_kubelet_node_ip=${2}
-k8s_linux_apiserver="ci/latest-${kubernetes_version}"
+k8s_linux_apiserver="stable-${kubernetes_version}"
```

Comment:
https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/cmd/kubeadm/app/util/version.go#L59

Reply:
good one, what's the version you are getting now ?

Reply:
the stable version of the minor.